synapse/api/room_versions.py

```diff
@@ -58,7 +58,12 @@ class RoomVersion(object):
     enforce_key_validity = attr.ib()  # bool
 
     # bool: before MSC2261/MSC2432, m.room.aliases had special auth rules and redaction rules
-    special_case_aliases_auth = attr.ib(type=bool, default=False)
+    special_case_aliases_auth = attr.ib(type=bool)
```

Comment:
I assume this change is because `default=False` is... the default?

Reply:
I removed the default because all of the room versions below are explicit about the value of this flag, so what's the point of having a default?